graph-subscriptions-api/src/network_subgraph.rs

```diff
@@ -63,6 +65,13 @@ impl SubgraphDeployments {
             .cloned()
             .unwrap_or_default()
     }
+    pub async fn subgraph(&self, subgraph_id: &SubgraphId) -> Option<Subgraph> {
+        let map = match self.inputs.value().await {
+            std::result::Result::Ok(map) => map,
+            Err(_) => return None,
+        };
```

Comment:
Fun trick: the `?` operator works on `Option<_>`
```suggestion
        let map = self.inputs.value().await.ok()?;
```